pyvisfile/vtk/vtk_ordering.py

```diff
@@ -59,13 +64,8 @@ def add_tuple_to_list(ary, x):
 
 # {{{ VTK_LAGRANGE_${SIMPLEX} (i.e. CURVE/TRIANGLE/TETRAHEDRON)
 
-def vtk_lagrange_curve_node_tuples(order, is_consistent=False):
-    if is_consistent:
-        node_tuples = [(0,), (order,)] + [(i,) for i in range(1, order)]
-    else:
-        node_tuples = [(i,) for i in range(order + 1)]
-
-    return node_tuples
+def vtk_lagrange_curve_node_tuples(order):
+    return [(0,), (order,)] + [(i,) for i in range(1, order)]
```

Comment:
To add some more explanations here. The `is_consistent` stuff was removed because it turned out to be a bug in the `vtkCellTypeSource` function I was using to check if we have the same order and not in the reader/writer/element definition.